### ISSUE TITLE
emc and item gone

### ISSUE THREAD

**Issue by skiller8641** [sinkillerj/ProjectE#2171]
Exact ProjectE version (do not say "latest", "latest on Curse", or similar): ProjectE-1.16.5-PE1.0.1B

Exact Forge version:36.2.21

Steps to reproduce:

1. be on cracked/online-mode=false
2. learn something
3. log out
4. log in

What I expected to happen:have my emc and object known in the table

What happened instead:nothing, it was empty

**Comment by chunye666** [comment]
I have the same problem, but my server version is 1.12.2 [CatServer-a8b73e9-universal] and the module version is "ProjectE-1.12.2-PE1.4.1", my players learn items in the conversion table, then log out of the server, then log back in, the items and EMC values in the conversion table all disappear. 
[Here](https://paste.liumingye.cn/5048) are the server debug log files for the related login and logout process.

**Comment by chunye666** [comment]
My problem has been solved with a little trick. My players only need to kill themselves once after logging into the server to have their previously learned items and EMC values restored.

**Comment by skiller8641** [comment]
yeah but after you loose your xp but what do you choose between 10 trillions EMC and 20 lv?

**Comment by chunye666** [comment]
Both. Because my server's gamerule of "keepInventory" is true. XDDDDDDD
